src/Redirect.tsx

```diff
@@ -32,4 +38,10 @@ if (__DEV__) {
   (Redirect.prototype as any).isReactComponent = {};
 }
 
-export default Redirect;
+// This actually doesn't extend a React.Component, but we need consumer to think that it does
+class RedirectType extends React.Component<RedirectOptions> {
+  // @ts-ignore
+  constructor(config: RedirectOptions);
+}
```

Comment:
I don't know whether there's a better way to do this. The problem is i need consumer to think this component `extends React.Component`

Reply:
I think this is ok but maybe do: 

```tsx
declare class RedirectType extends React.Component<RedirectOptions> {
  constructor(config: RedirectOptions);
}
```

with the declare which should make it just a type?

Reply:
yep, it works :smile: